_episodes/01-connecting.md

```diff
@@ -50,6 +50,17 @@ MS-DOS (Microsoft Disk Operating System) it does not have an SSH tool built into
 to be installed. There are a variety of programs that can be used for this; two common ones we
 describe here, as follows:
 
+#### Git BASH
+
+Git BASH gives you a terminal like interface in Windows. You can use this to connect to a remote 
+computer by SSH. It is can be downloaded for free from
```

Comment:
```suggestion
computer with `ssh`. It can be downloaded for free from
```

Reply:
Would love a native speaker's comment on this, @tkphd @ocaisa @reid-a!

Reply:
SSH rather than `ssh` seems to be used in this lesson - am ok with either, but it would be good to be consistent.

Reply:
is should be removed

Reply:
I agree that SSH is used elsewhere. I usually say `via SSH`, but I think that might be nitpicking.

Reply:
Removed "is", kept "SSH", used "via SSH" 

Thanks for the suggestions